x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/function/scalar/datetime/CurrentDateTimeTests.java

```diff
@@ -39,8 +49,8 @@ protected CurrentDateTime copy(CurrentDateTime instance) {
     protected CurrentDateTime mutate(CurrentDateTime instance) {
         ZonedDateTime now = instance.configuration().now();
         ZoneId mutatedZoneId = randomValueOtherThanMany(o -> Objects.equals(now.getOffset(), o.getRules().getOffset(now.toInstant())),
-                () -> randomZone());
-        return new CurrentDateTime(instance.source(), Literal.of(EMPTY, randomInt(10)), TestUtils.randomConfiguration(mutatedZoneId));
+            ESTestCase::randomZone);
+        return new CurrentDateTime(instance.source(), Literal.of(EMPTY, randomInt(9) + 1), TestUtils.randomConfiguration(mutatedZoneId));
```

Comment:
Why choosing only precision >= 1?
And, in any case, there is a method `randomIntBetween` for a range of values (`randomIntBetween(1, 10)`).

Reply:
thx, rushed here and didn't check the options.

Reply:
actually the fix is just `randomInt(9)` as it is inclusive meaning [0, 9]. 10 is not valid anymore as the precision range is now checked in the functions.